scripts/run-ci-e2e-tests.js

```diff
@@ -70,6 +71,33 @@ try {
 
   const REACT_NATIVE_PACKAGE = path.join(ROOT, 'react-native-*.tgz');
 
+  describe('Set up Verdaccio');
+  const verdaccioProcess = spawn('npx', [
+    'verdaccio@5.15.3',
+    '--config',
+    '.circleci/verdaccio.yml',
+  ]);
+  VERDACCIO_PID = verdaccioProcess.pid;
+  exec('npx wait-on@6.0.1 http://localhost:4873');
+  exec('npm set registry http://localhost:4873');
+  exec('echo "//localhost:4873/:_authToken=secretToken" > .npmrc');
+
+  describe('Publish packages');
+  const packages = JSON.parse(
+    JSON.parse(exec('yarn --json workspaces info').stdout).data,
+  );
+  Object.keys(packages)
+    .filter(
+      packageName =>
+        packageName !== '@react-native/tester' &&
```

Comment:
Now `'@react-native/tester'` is private, so perhaps this is not needed anymore?

Reply:
good point, we should be able to skip that.